BackEnd/BackEndBigChallenge/app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\LoginRequest;
+use App\Models\User;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Facades\Hash;
+
+class LoginController extends Controller
+{
+    public function __invoke(LoginRequest $request): JsonResponse
+    {
+        $arguments = $request->validated();
+        $user = User::where('email', $arguments['email'])->first();
+        if (!$user || !Hash::check($arguments['password'], $user->password)) {
+            return response()->json([
```

Comment:
Otra opcion para retornar el mensaje de error es usar:
`throw ValidationException::withMessages([
                'email' => ['The provided credentials are incorrect.'],
            ]);`